### PR TITLE
add author mapping precompile docs

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This PR adds a new page for the author mapping precompile. It also updates the [Collator Account Management](https://docs.moonbeam.network/node-operators/networks/collators/account-management/) page.

### Checklist

- [x] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [ ] If pages have been moved around, I have run the `move-pages.py` script to move the pages and update the image paths on the chinese repo
    - [ ] After the script has been run, I have created an additional PR in `moonbeam-docs-cn`
- [ ] If images have been added, I have run the `compress-images.py` script to compress the images.
- [ ] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables
- [ ] If this page requires a disclaimer, I have added one

### Corresponding PRs

N/A 

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [x] No additional PRs are required after the translations are done

#### Items to be Updated

N/A

### CODE REVIEW

File: builders/pallets-precompiles/precompiles/author-mapping.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+keywords: solidity, ethereum, author mapping, collator, moonbeam, precompiled, contracts, block producer
+---
+
+# Interacting with the Author Mapping Precompile
+
+![Precomiled Contracts Banner](/images/builders/pallets-precompiles/precompiles/author-mapping/author-mapping-banner.png)
+
+## Introduction {: #introduction } 
+
+The author mapping precompiled contract on Moonbeam allows collator candidates to map session keys to a Moonbeam address, where block rewards are paid out, through a familiar and easy-to-use Solidity interface.
```
Comment (albertov19) [inline builders/pallets-precompiles/precompiles/author-mapping.md:13]: Should we say that this allows them to run a collator using Ledger?

File: builders/pallets-precompiles/precompiles/author-mapping.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+1. Click on the **File explorer** tab
+2. Copy and paste the file contents into a [Remix file](https://remix.ethereum.org/){target=_blank} named `AuthorMappingInterface.sol`
+
+![Copying and Pasting the Author Mapping Interface into Remix](/images/builders/pallets-precompiles/precompiles/author-mapping/author-mapping-1.png)
+
+### Compile the Contract {: #compile-the-contract } 
+
+1. Click on the **Compile** tab, second from top
+2. Then to compile the interface, click on **Compile Proxy.sol**
```
Comment (albertov19) [inline builders/pallets-precompiles/precompiles/author-mapping.md:117]:
```suggestion
2. Then to compile the interface, click on **Compile AutorMappingInterface.sol**
```

**Review by albertov19 — CHANGES_REQUESTED** [review]
LGTM - Left some minor comments

File: builders/pallets-precompiles/precompiles/author-mapping.md (hunk 1 above)
Comment (eshaben) [reply builders/pallets-precompiles/precompiles/author-mapping.md:13]: It's in the following paragraphs - should I move it up to this first sentence/paragraph?

File: builders/pallets-precompiles/precompiles/author-mapping.md (hunk 2 above)
Comment (eshaben) [reply builders/pallets-precompiles/precompiles/author-mapping.md:117]: oops thanks! haha

Comment (eshaben) [inline builders/pallets-precompiles/precompiles/author-mapping.md:117]:
```suggestion
2. Then to compile the interface, click on **Compile AuthorMappingInterface.sol**
```

File: builders/pallets-precompiles/precompiles/author-mapping.md (hunk 1 above)
Comment (albertov19) [reply builders/pallets-precompiles/precompiles/author-mapping.md:13]: Sure. I feel we should show the main value of this precompile a bit earlier

File: builders/pallets-precompiles/precompiles/author-mapping.md [hunk 3]
```diff
@@ -10,12 +10,10 @@ keywords: solidity, ethereum, author mapping, collator, moonbeam, precompiled, c
 
 ## Introduction {: #introduction } 
 
-The author mapping precompiled contract on Moonbeam allows collator candidates to map session keys to a Moonbeam address, where block rewards are paid out, through a familiar and easy-to-use Solidity interface.
+The author mapping precompiled contract on Moonbeam allows collator candidates to map session keys to a Moonbeam address, where block rewards are paid out, through a familiar and easy-to-use Solidity interface. This enables candidates to complete author mapping with a Ledger or Ethereum-compatible wallet. However, it is recommended to generate your keys on an air-gapped machine. You can find out more information by referring to the [account requirements section of the Collator Requirements page](/node-operators/networks/collators/requirements/#account-requirements){target=_blank}.
```
Comment (albertov19) [inline builders/pallets-precompiles/precompiles/author-mapping.md:13]: "or any other Ethereum wallet compatible with Moonbeam"?

**Review by albertov19 — APPROVED** [review]
LGTM

File: builders/pallets-precompiles/precompiles/author-mapping.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+keywords: solidity, ethereum, author mapping, collator, moonbeam, precompiled, contracts, block producer
+---
+
+# Interacting with the Author Mapping Precompile
+
+![Precomiled Contracts Banner](/images/builders/pallets-precompiles/precompiles/author-mapping/author-mapping-banner.png)
+
+## Introduction {: #introduction } 
+
+The author mapping precompiled contract on Moonbeam allows collator candidates to map session keys to a Moonbeam address, where block rewards are paid out, through a familiar and easy-to-use Solidity interface. This enables candidates to complete author mapping with a Ledger or Ethereum-compatible wallet. However, it is recommended to generate your keys on an air-gapped machine. You can find out more information by referring to the [account requirements section of the Collator Requirements page](/node-operators/networks/collators/requirements/#account-requirements){target=_blank}.
```
Comment (eshaben) [inline builders/pallets-precompiles/precompiles/author-mapping.md:13]:
```suggestion
The author mapping precompiled contract on Moonbeam allows collator candidates to map session keys to a Moonbeam address, where block rewards are paid out, through a familiar and easy-to-use Solidity interface. This enables candidates to complete author mapping with a Ledger or any other Ethereum wallet compatible with Moonbeam. However, it is recommended to generate your keys on an air-gapped machine. You can find out more information by referring to the [account requirements section of the Collator Requirements page](/node-operators/networks/collators/requirements/#account-requirements){target=_blank}.
```

**Pull request merged**